### DISCUSSION TITLE
Samsung S24 to f36 with NBT connect solved

### DISCUSSION THREAD

**Discussion by DVEL3601** [BimmerGestalt/AAIdrive#820] [Show and tell]
I switched ricently to a S24 Samsung and was not able to connect (previous Huawei P30 no issues). I tried different cables etc. But no help to connect , even the phone was not charging properly from the usb . 

Finally managed to connect with the original USB-C cable to USB-C cable , using a simple Aliexpress small USB to USB-C adapter . Still have to connect to the car before you start the engine and additionally have manually LogInto MyBMW .

Great App guys i will donate again , keep up !

**Comment by hufman** [comment]
Glad to hear it! USB connectivity is flaky, unfortunately, but it sounds like you figured out a ritual that works for you!
